backend/src/main/resources/application.yaml

```diff
@@ -78,6 +78,7 @@ management:
   endpoint.health.probes.enabled: true
   endpoint.info.enabled: true
   endpoints.web.exposure.include: health, info
+  endpoint.health.show-components: always
```

Comment:
Need this tweak since the new actuator endpoint is a component of the `health` endpoint. Without it, the API endpoint doesn't load